### PR TITLE
Big update channels 2021.09

### PR DESCRIPTION
**Pull request by taodicakhia**
Update:
- Add few channels
- Delete (Replace) some channels
- Change iptv query url

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
I'll let you tell us when you've finished, so that we can review it once the PR is complete. Tell us once it's good for review.

File: channels/al.m3u [hunk 1]
```diff
@@ -13,8 +13,8 @@ https://shkoder.gjirafa.com/api/media/rgjirafa/t0110y/index.m3u8
 http://93.157.62.180/Cufo/index.m3u8
 #EXTINF:-1 tvg-id="FrameTV.al" tvg-country="AL" tvg-language="" tvg-logo="" group-title="",Frame TV (720p) [Not 24/7]
 http://195.154.252.221:8000/play/a002/index.m3u8
-#EXTINF:-1 tvg-id="KLAN.al" tvg-country="AL" tvg-language="" tvg-logo="https://i.imgur.com/pWoMc0r.png" group-title="",KLAN [Offline]
-http://79.106.73.244:4040/live/klanhdmob/playlist.m3u8
+#EXTINF:-1 tvg-id="KLAN.al" tvg-country="AL" tvg-language="" tvg-logo="https://i.imgur.com/pWoMc0r.png" group-title="",KLAN (720p)
+http://93.157.62.180/KlanPlus/index.m3u8
```
Comment (LaneSh4d0w) [inline channels/al.m3u:17]: This stream is Klan Plus, not Klan.

Comment (taodicakhia) [reply channels/al.m3u:17]: But content of channel belongs Klan

Comment (LaneSh4d0w) [reply channels/al.m3u:17]: Klan Plus and Klan are two distinct channels, if you look at the playlist, which is why i'm saying it.

File: channels/unsorted.m3u [hunk 2]
```diff
@@ -503,10 +477,7 @@ https://ov.ottera.tv/live/master.m3u8?channel=mcom_gv_us
 https://simultv.s.llnwi.net/o059/Thrillerz/interlink.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Tide TV
 https://5889e7d0d6e28.streamlock.net/tide-live/_definst_/smil:livestream.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Top Channel
-http://us.bestvideostreaming.is/8708/index.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Top News
-http://93.157.62.180/TopNews/index.m3u8
+
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:480]: White space on line 480.

File: channels/al.m3u (hunk 1 above)
Comment (taodicakhia) [reply channels/al.m3u:17]: ok content of channel is klan plus

File: channels/unsorted.m3u [hunk 3]
```diff
@@ -391,36 +363,6 @@ https://58bd5b7a98e04.streamlock.net/medienasa-live/_definst_/mp4:RFH_high/playl
...
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Plus
-http://79.106.48.2:4578/live/rtsh_plus_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Sat
-http://79.106.48.2:4578/live/rtsh_sat_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Shkodra
-http://79.106.48.2:4578/live/rtsh_shkodra_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Shqip
-http://79.106.48.2:4578/live/rtsh_shqip_mob_p1/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",RTSH Sport
-http://79.106.48.2:4578/live/sport_mob_ppp1/playlist.m3u8
```
Comment (freearhey) [inline channels/unsorted.m3u:423]: The links from 79.106.48.2 are currently working
vlcsnap-2021-09-30-22h04m10s573

File: channels/dz.m3u [hunk 4]
```diff
@@ -1,49 +1,7 @@
...
-#EXTINF:-1 tvg-id="ElBilad.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/Bp00gMu.png" group-title="",El Bilad [Offline]
-https://cdn02.hta.dz/abr_htatv/EL_BILAD/playlist.m3u8
-#EXTINF:-1 tvg-id="ElDjazairN1.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/EoB4f6e.png" group-title="",El Djazair N1 [Offline]
-https://cdn02.hta.dz/abr_htatv/El_Djazair_N1/playlist.m3u8
-#EXTINF:-1 tvg-id="ElDjazairiaOne.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/OhTioAg.png" group-title="",El Djazairia One [Not 24/7]
-https://cdn02.hta.dz/abr_htatv/EL_DJAZAIRIA_TV/playlist.m3u8
-#EXTINF:-1 tvg-id="ElFadjrTV.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/Br7bAVR.png" group-title="",El Fadjr TV [Offline]
-https://cdn02.hta.dz/abr_htatv/EL_FADJR_TV_DZ/playlist.m3u8
-#EXTINF:-1 tvg-id="ElHayatTV.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/fNLiJwK.png" group-title="",El Hayat TV [Offline]
-https://cdn02.hta.dz/abr_htatv/EL_HAYAT_TV_ALGERIE/playlist.m3u8
```
Comment (freearhey) [inline channels/dz.m3u:33]: We do not remove broken links from playlists if they have no alternatives to keep the channel description (logo, language, country)

File: channels/dz.m3u [hunk 5]
```diff
@@ -1,49 +1,7 @@
...
-#EXTINF:-1 tvg-id="AlgerieTV5.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/gXG15AX.png" group-title="",Algérie TV5 [Offline]
-https://cdn02.hta.dz/abr_htatv/TV_5/playlist.m3u8
-#EXTINF:-1 tvg-id="AlgerieTV6.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/HYqJY6y.png" group-title="",Algérie TV6 [Offline]
-https://cdn02.hta.dz/abr_htatv/TV_6_HD/playlist.m3u8
-#EXTINF:-1 tvg-id="BahiaTV.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/oBRXuH2.png" group-title="",Bahia TV [Offline]
-https://cdn02.hta.dz/abr_htatv/Bahia_TV/playlist.m3u8
-#EXTINF:-1 tvg-id="CanalAlgerie.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/FPmDhNH.png" group-title="",Canal Algérie [Offline]
-https://cdn02.hta.dz/abr_htatv/CANAL_ALGERIE/playlist.m3u8
-#EXTINF:-1 tvg-id="CNA.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/4s1NlRf.jpg" group-title="Religious",CNA (Chaîne Nord Africaine) (360p)
-https://live.creacast.com/cna/smil:cna.smil/playlist.m3u8
```
Comment (freearhey) [inline channels/dz.m3u:17]: We do not remove broken links from playlists if they have no alternatives to keep the channel description (logo, language, country)

File: channels/dz.m3u [hunk 6]
```diff
@@ -1,49 +1,7 @@
...
-#EXTINF:-1 tvg-id="ENTV.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/tFXvY71.png" group-title="",ENTV [Offline]
-https://cdn02.hta.dz/abr_htatv/PROGRAMME_NATIONAL/playlist.m3u8
-#EXTINF:-1 tvg-id="LinaTV.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/WeGG33J.png" group-title="",Lina TV [Offline]
-https://cdn02.hta.dz/abr_htatv/Lina_TV/playlist.m3u8
-#EXTINF:-1 tvg-id="SamiraTV.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/5C80xDT.png" group-title="",Samira TV [Offline]
-https://cdn02.hta.dz/abr_htatv/SamiraTV/playlist.m3u8
-#EXTINF:-1 tvg-id="TV7Elmaarifa.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/dYlAfFZ.png" group-title="",TV7 Elmaarifa [Offline]
-https://cdn02.hta.dz/abr_htatv/TV7_ELMAARIFA/playlist.m3u8
-#EXTINF:-1 tvg-id="TV8Edhakira.dz" tvg-country="DZ" tvg-language="Arabic" tvg-logo="https://i.imgur.com/l7hw1W6.png" group-title="",TV8 Edhakira [Offline]
-https://cdn02.hta.dz/abr_htatv/TV8_EDHAKIRA/playlist.m3u8
```
Comment (freearhey) [inline channels/dz.m3u:49]: We do not remove broken links from playlists if they have no alternatives to keep the channel description (logo, language, country)

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

File: channels/unsorted.m3u (hunk 3 above)
Comment (taodicakhia) [reply channels/unsorted.m3u:423]: But the channels are determined

Comment (freearhey) [reply channels/unsorted.m3u:423]: I'm sorry, but I don't understand what you mean.

Comment (LaneSh4d0w) [reply channels/unsorted.m3u:423]: They might work only for certain IPs, doesn't work for me from where I am right now.

File: channels/id.m3u [hunk 7]
```diff
@@ -28,21 +28,21 @@ http://iamchannel.org:1935/tes/1/playlist.m3u8
 #EXTINF:-1 tvg-id="IndonesiaChannel.id" tvg-country="ID" tvg-language="" tvg-logo="" group-title="",Indonesia Channel (720p) [Not 24/7]
 http://wowzaprod236-i.akamaihd.net/hls/live/1019903/7dd4cf51/playlist.m3u8
 #EXTINF:-1 tvg-id="Indosiar.id" tvg-country="ID" tvg-language="Indonesian" tvg-logo="" group-title="General",Indosiar [Offline]
-https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.vidio.com/live/205-indosiar
+https://iptv.dc-team.com/query?url=https://www.vidio.com/live/205-indosiar
```
Comment (MinePlayersPE) [inline channels/id.m3u:31]: Not sure if it's just me, but I'm getting 504's when trying to access these Vidio links (either from browser, VLC or ffplay)

File: channels/unsorted.m3u (hunk 3 above)
Comment (taodicakhia) [reply channels/unsorted.m3u:423]: The channels is same as in al.m3u, so I remove them

Comment (freearhey) [reply channels/unsorted.m3u:423]: @taodicakhia We try to keep all versions of the channel broadcast in case one of them fails.

Comment (taodicakhia) [reply channels/unsorted.m3u:423]: @freearhey ok

**Comment by taodicakhia** [comment]
All are ok

File: channels/ae.m3u [hunk 8]
```diff
@@ -1,7 +1,7 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="3eeshAlAanTV.ae" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="https://www.3eeshalaan.net/wp-content/themes/alaan-child-3eesh/assets/img/logo.png" group-title="General",3eesh Al Aan TV (720p)
 https://streaming.3eeshalaan.net/AAAFinalFeed/AlAanFeed_live.m3u8
-#EXTINF:-1 tvg-id="AbuDhabiAloula.ae" tvg-country="AE" tvg-language="Arabic" tvg-logo="" group-title="",Abu Dhabi Aloula (1080p)
+#EXTINF:-1 tvg-id="AbuDhabiAloula.ae" tvg-country="AE" tvg-language="Arabic" tvg-logo="" group-title="",Abu Dhabi TV (1080p)
```
Comment (freearhey) [inline channels/ae.m3u:4]:
```suggestion
#EXTINF:-1 tvg-id="AbuDhabiTV.ae" tvg-country="AE" tvg-language="Arabic" tvg-logo="" group-title="",Abu Dhabi TV (1080p)
```

File: channels/af.m3u [hunk 9]
```diff
@@ -11,8 +11,6 @@ https://d2g7v53450s2i2.cloudfront.net/ATNUS/streamdelay/playlist.m3u8
 https://d10rltuy0iweup.cloudfront.net/ATNNEWS/myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="BaharTV.af" tvg-country="AF" tvg-language="" tvg-logo="" group-title="",Bahar TV (720p)
 https://59d39900ebfb8.streamlock.net/bahartv/bahartv/playlist.m3u8
-#EXTINF:-1 tvg-id="BaryaTV.af" tvg-country="AF" tvg-language="" tvg-logo="https://i.imgur.com/yf5cI8r.jpg" group-title="",Barya TV (720p) [Offline]
-http://51.210.199.56/hls/stream.m3u8
```
Comment (freearhey) [inline channels/af.m3u:15]: It's better to leave it on the playlist, too

File: channels/af.m3u [hunk 10]
```diff
@@ -25,11 +23,17 @@ http://208.93.117.113/live/Stream1/playlist.m3u8
 https://playout395.livestreamingcdn.com/live/Stream1/playlist.m3u8
 #EXTINF:-1 tvg-id="PamirTV.af" tvg-country="AF" tvg-language="" tvg-logo="" group-title="",Pamir TV (1080p)
 http://live.stream.cdn.pamirtv.com/ptv/d0dbe915091d400bd8ee7f27f0791303.sdp/index.m3u8
-#EXTINF:-1 tvg-id="Sharq.af" tvg-country="AF" tvg-language="Pashto" tvg-logo="https://ws.shoutcast.com/images/contacts/b/b9f8/b9f811c5-d210-4d0b-ae32-f467823a913e/radios/0d677ea5-46b4-4129-9359-9e5783fb6a37/0d677ea5-46b4-4129-9359-9e5783fb6a37.png" group-title="General",Sharq (576p) [Offline]
-http://51.210.199.50/hls/stream.m3u8
```
Comment (freearhey) [inline channels/af.m3u:29]: Same

File: channels/af.m3u [hunk 11]
```diff
@@ -25,11 +23,17 @@ http://208.93.117.113/live/Stream1/playlist.m3u8
...
 #EXTINF:-1 tvg-id="SolhTV.af" tvg-country="AF" tvg-language="" tvg-logo="https://i.imgur.com/bAbNyIt.jpg" group-title="",Solh TV (576p)
 http://51.210.199.42/hls/stream.m3u8
 #EXTINF:-1 tvg-id="ToloTV.af" tvg-country="AF" tvg-language="Pashto" tvg-logo="https://pbs.twimg.com/profile_images/579001245697908737/bSnXEBFo_400x400.jpeg" group-title="",Tolo TV (720p)
 https://raw.githubusercontent.com/taodicakhia/IPTV_Exception/master/channels/af/tolotv.m3u8
-#EXTINF:-1 tvg-id="Tuti.af" tvg-country="AF" tvg-language="Pashto" tvg-logo="https://i.imgur.com/JerfIBt.jpg" group-title="",Tuti (480p) [Not 24/7]
-https://rrsatrtmp.tulix.tv/livecdn827/myStream.sdp/playlist.m3u8
```
Comment (freearhey) [inline channels/af.m3u:35]: Same

File: channels/az.m3u [hunk 12]
```diff
@@ -29,6 +29,10 @@ https://str2.yodacdn.net/publive/ftv/video.m3u8
 https://node19.connect.az:8086/ch1/ch1.hi.m3u8
 #EXTINF:-1 tvg-id="IctimaiTV.az" tvg-country="AZ" tvg-language="Azerbaijani;Russian" tvg-logo="https://itv.az/assets/images/itv-logo.png" group-title="General",İctimai TV (480p)
 https://node19.connect.az:8086/ch1/ch1.med.m3u8
+#EXTINF:-1 tvg-id="ITV.az" tvg-country="AZ" tvg-language="Azerbaijani" tvg-logo="https://www.lyngsat-stream.com/logo/tv/ii/ictimai-tv-az.png" group-title="",İTV
+https://node19.connect.az:9081/tv/live/playlist.m3u8
```
Comment (freearhey) [inline channels/az.m3u:33]: image

File: channels/az.m3u [hunk 13]
```diff
@@ -29,6 +29,10 @@ https://str2.yodacdn.net/publive/ftv/video.m3u8
...
+#EXTINF:-1 tvg-id="QBLTV.az" tvg-country="AZ" tvg-language="" tvg-logo="" group-title="",QƏBƏLƏ TV (480p)
+https://qebele.tv/live/stream/index.m3u8
```
Comment (freearhey) [inline channels/az.m3u:35]: The link is already in the playlist.

File: channels/af.m3u (hunk 9 above)
Comment (taodicakhia) [reply channels/af.m3u:15]: Ok

File: channels/af.m3u (hunk 10 above)
Comment (taodicakhia) [reply channels/af.m3u:29]: Ok

**Comment by taodicakhia** [comment]
All are ok

File: channels/pe.m3u [hunk 14]
```diff
@@ -301,5 +313,8 @@ https://5ee0faac3bbae.streamlock.net/visionnoticias/visionnoticias/playlist.m3u8
 https://5ee0faac3bbae.streamlock.net/visionmusica/visionmusica/playlist.m3u8
 #EXTINF:-1 tvg-id="Willax.pe" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/willaxtv/picture?width=320&height=320" group-title="General",Willax (480p)
 https://iptv.dc-team.com/query?url=https://www.dailymotion.com/WillaxTV
+http://free.fullspeed.tv/query?url=https://www.dailymotion.com/WillaxTV
```
Comment (LaneSh4d0w) [inline channels/pe.m3u:316]: Quack.

Comment (jkemp814) [reply channels/pe.m3u:316]: Line 315 was not deleted.

Comment (taodicakhia) [reply channels/pe.m3u:317]: Sorry because I had resolved conflicts

Comment (jkemp814) [reply channels/pe.m3u:317]: There is no pound sign before EXTINF:-1

Should be #EXTINF:-1

File: channels/fr.m3u [hunk 15]
```diff
@@ -255,6 +255,8 @@ https://live.creacast.com/mirabelletv/smil:mirabelletv.smil/playlist.m3u8
 https://streamer01.myvideoplace.tv/streamer02/hls/MDS_VIA_PAD_301117.m3u8
 #EXTINF:-1 tvg-id="W9.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://i.imgur.com/7FFFvCN.jpg" group-title="Entertainment",W9 (720p)
 http://flusonic-1.platinum-tv.com/W9/mono.m3u8?token=test
+#EXTINF:-1 tvg-id="WéoNordPasdeCalais.fr" tvg-country="FR" tvg-language="French" tvg-logo="" group-title="Local",Wéo (Nord-Pas-de-Calais) (480p) [Not 24/7]
+https://iptv.dc-team.com/query?url=https://www.dailymotion.com/video/x62islg
```
Comment (LaneSh4d0w) [inline channels/fr.m3u:259]: This one is the same than the "Wéo Hauts-de-France" link below.

Note that the Nord-Pas-de-Calais region is also inside the Hauts-de-France (Picardy, Nord-Pas-de-Calais).

**Comment by LaneSh4d0w** [comment]
The "iptv.dc-team.com" domain is meant to disappear very soon. Please move to the "http://free.full-speed.tv" domain.

**Comment by taodicakhia** [comment]
> The "iptv.dc-team.com" domain is meant to disappear very soon. Please move to the "http://free.fullspeed.tv" domain.

Updated

**Pull request closed without merging**